oak_functions/client/java/src/com/google/oak/functions/client/AttestationClient.java

```diff
@@ -214,4 +227,36 @@ public Response send(Request request)
     byte[] decryptedResponse = encryptor.decrypt(responsePayload);
     return Response.parseFrom(decryptedResponse);
   }
+
+  /**
+   * Intercepts gRPC requests and adds Metadata with an API key.
+   * https://cloud.google.com/endpoints/docs/grpc/restricting-api-access-with-api-keys#java
+   */
+  private static final class Interceptor implements ClientInterceptor {
+    private final String apiKey;
+
+    private static final Metadata.Key<String> API_KEY_HEADER =
+        Metadata.Key.of("x-goog-api-key", Metadata.ASCII_STRING_MARSHALLER);
```

Comment:
Where does this string come from? Please add links / documentation, and consider making a constant.

Reply:
Done